source/features/sticky-sidebar.tsx

```diff
@@ -16,11 +16,12 @@ function updateStickiness(): void {
 }
 
 const onResize = debounce(updateStickiness, {wait: 100});
+const observer = new ResizeObserver(onResize);
 
 function init(): void {
 	observe(sidebarSelector, {
 		add(sidebar) {
```

Comment:
Why does this not have a "not selector"

Reply:
It’s not making any changes. We’re just listening to it. ResizeObserver already deduplicates identical .observe calls, I suppose

Reply:
ok